src/main.tsx

```diff
@@ -14,6 +15,7 @@ ReactDOM.createRoot(document.getElementById('root')!).render(
       <Route element={<Layout />}>
         <Route path="/avatar-stack" element={<AvatarStack />} />
         <Route path="/emoji-reactions" element={<EmojiReactions />} />
+        <Route path="/user-claims" element={<Claims />} />
```

Comment:
We need to rename this tp `<UserClaims/>

Reply:
Done https://github.com/ably-labs/realtime-examples/pull/37/commits/b2f0c9cc51aa0f6156b88db5613df8c6f313a9ae